data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/State.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_impl;
+
+import java.util.Optional;
+
+/**
+ * Business object for a state (instant and contingency) in the CRAC file.
+ *
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ */
+public class State {
```

Comment:
Shouldn't we add an attribute: list of AbstractUsageRule?

Reply:
We'll do it later if needed.